tools/travis/install_qt.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+set -ex
+
+if [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
+    echo "backend : Template" > $MPL_DIR/matplotlibrc
+fi
+# Now configure Matplotlib to use Qt4
+if [[ "${QT}" == "PyQt4" ]]; then
```

Comment:
I vote for deprecating Qt4 altogether

Reply:
Actually deprecating is the wrong word. Removing. Nuking. Obliterating. =P

Reply:
Yeah probably best. I just didn't want to impose things you know.

Reply:
Absolutely. For giant changes like this one, sometimes you can ask the mailing list or on the PR directly about what to do. It sometimes helps to avoid unnecessary work. But you have pushed things along very quickly by just forging ahead, so I very much appreciate that! =)

Reply:
Do you mean use something like `qtpy`?

Reply:
One data point on this is that riverbank have pulled all the pyqt4 files from pypi (https://pypi.org/project/PyQt4/).

Reply:
@tacaswell , it seems that conda shipped pyqt5.6 for 2.7 but not pyqt5.9.
Seeing as 0.14 is the last release to officially support 2.7, I think we are good :D.

Reply:
If Riverbank pulled 4, then 5-only seems very reasonable.